style.css

```diff
@@ -0,0 +1,193 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr;
+  grid-template-rows: minmax(100px, auto);
+  align-items: center;
+}
+
+.nav .logo {
+  height: 36px;
+}
+
+.nav > div {
+  padding: 1em;
+}
+
+/* searchbar */
+.nav * {
+  box-sizing: border-box;
+}
+
+/* Style the search field */
+.nav form.searchbar input[type="text"] {
+  padding: 10px;
+  font-size: 17px;
+  border: 1px solid grey;
+  float: left;
+  width: 80%;
+  background: #f1f1f1;
+}
+
+/* Style the submit button */
+.nav form.searchbar button {
+  float: left;
+  width: 20%;
+  padding: 10px;
+  background: #e4e4e4;
+  color: rgb(104, 104, 104);
+  font-size: 17px;
+  border: 1px solid grey;
+  border-left: none; /* Prevent double borders */
+  cursor: pointer;
+}
+
+.nav form.searchbar button:hover {
+  background: #cccaca;
+}
+
+/* Clear floats */
+.nav form.searchbar::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+/* Navigation Icons */
+.nav .icons {
+  justify-self: end;
+  color: #606060;
+}
+
+.nav .icons i {
+  padding-right: 16px;
+}
+
+/* Main Section */
+.main {
+  display: flex;
+  justify-content: space-between;
+}
+
+.main .video {
+  flex-basis: 75%;
+}
+
+.main .video iframe {
+  width: 100%;
+  height: 500px;
+  border: none;
+}
+
+.main .aside {
+  flex-basis: 23%;
+}
+
+.main .aside .item {
+  display: flex;
+}
+
+.main .aside .item div {
+  flex: 1;
+}
+
+.video .details-2 img {
+  padding: 1em;
+  border-radius: 45px;
+}
+
+.main .aside .item img {
+  width: 100%;
+  height: 85%;
+}
+
+.main .aside .item .vid_info {
+  display: flex;
+  flex-direction: column;
+  margin-left: 7px;
+}
+
+.main .aside .vid_info p {
+  margin: 0 0 2px 0;
+  padding: 0;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  color: #9b9b9b;
+  font-size: 14px;
+
+}
+
+.main .aside .vid_info p:first-child {
+  font-weight: bold;
+  color:black;
+  font-size:16px;
+}
+
+/* Video description */
+
+.video .desc .details {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+  border-bottom: 1px solid #ccc;
+  color: #606060;
+}
+
+.video .desc h1 {
+  font-size: 26px;
+}
+
+.video .desc .details .buttons {
+  text-align: right;
+}
```

Comment:
- [ ] The previous TSE made an observation here and will like to clarify it. When you have classes, there is not necessary to create the "path" that leads to the element you want to style. In this code, are you saying that you want the elements with the vide, desc, details and buttons classes to have the text aligned to the right? or are you saying that you want the text inside the buttons to be aligned to the right?

If the answer is yes to the first question, you are doing it right. If your answer is no, then you just need the buttons class in there and delete the rest. That's what classes are for.

You would need to specify such path when using tags, for example, when you want to style the main > div > buttons.

Make sure you are actually styling what you want and if necessary, make the changes in all cases.